cleanlab/models/keras.py

```diff
@@ -82,6 +82,7 @@ def __init__(
         self.net = None
 
     def get_params(self, deep=True):
+        """Returns the parameters of the Keras classifier."""
```

Comment:
[nit] should we rewrite "Keras classifier" -> "Keras model" everywhere here to make it clearer these are the exact same parameters/methods as for a standard Keras model?  (I'm not sure if Keras docs ever call their model a "classifier" but if so then this seems ok as is).

Reply:
Keras model is more commonly used, will change accordingly